### PR TITLE
added endpoint to restaurant route

### LINKED ISSUE #81: Create API function - get all bookings for a restaurant
**User Story:**
As a restaurant manager, I want to be able to view all bookings so that I can make decisions on how much staff I need.

**Acceptance Criteria:**

* [ ] Created a function that gets all available booking information for a restaurant. 

**Additional Context:**
Will need a way to delete bookings automatically after booking date/time has been surpassed

### PR DESCRIPTION
**Pull request by ljcnz**
## Link to Issue Number
closes #81

## Description
I've added a new endpoint to get all reservations for a given restaurant. Returns a JSON list of reservations. Takes in a restaurant id in the url in the form /restaurants/{restaurantID}/reservations

## Checklist
- [x] All feature request A/C have been met OR the bug has been fixed
- [x] I ran all necessary tests and they pass
- [x] I rebased upstream/master onto my working branch before opening this PR
- [x] I have named my PR something sensible
- [x] I have included the issue number above
- [x] I have assigned at least two people to review my changes

## Other Notes
No tests created.
Made a swagger docstring

### CODE REVIEW

File: backend/routes/restaurant.js [hunk 1]
```diff
@@ -158,4 +158,42 @@ router.delete('/', (req, res) => {
...
+router.get('/:restaurantID/reservations', (req, res) => {
+  const id = req.params.restaurantID;
+
+  if (!id) {
+    res.status(400).json({ error: 'path param: {id} malformed' });
+    return;
+  }
+
+  connection.query(
+    `SELECT * FROM RESERVATION WHERE RestaurantID = ${id}`,
```
Comment (merw957) [inline backend/routes/restaurant.js]: This should use a placeholder ('SELECT * FROM RESERVATION WHERE RestaurantID = ?', [id]) to prevent SQL injection.

Comment (ljcnz) [reply backend/routes/restaurant.js]: I've addressed this in 3df09e9

**Comment by ljcnz** [comment]
While working on this issue, eric and I hadn't realized a similar thing had been done in the reservations endpoint. Becuase the previous end point wasn't created in the right place, and didn't abide standard REST conventions, I've removed the old api and the one I initially created will remain the standard.

**Comment by MingChen91** [comment]
Need to resolve conflicts which just seems to be the removal of the original endpoint. 
Could you just copy the original test and change the endpoint?

File: backend/routes/restaurant.js [hunk 2]
```diff
@@ -158,4 +158,42 @@ router.delete('/', (req, res) => {
...
+ *         type: integer
+ *     responses:
+ *       200:
+ *         description: Successfully retrieved
+ */
+router.get('/:restaurantID/reservations', (req, res) => {
+  const id = req.params.restaurantID;
+
+  if (!id) {
+    res.status(400).json({ error: 'path param: {id} malformed' });
```
Comment (jacoballen4534) [inline backend/routes/restaurant.js]: I believe these should be backticks and a "$" in front of the "{id}"

Comment (ljcnz) [reply backend/routes/restaurant.js]: Oops thanks for catching that

**Comment by ljcnz** [comment]
Don't merge just yet, still awaiting the results of that uni testing conversation.

**Comment by ljcnz** [comment]
Closing this pull request as the issue will now be addressed as part of a wider-scale issue in #109

**Pull request closed without merging**